formbricks/formbricks
# fix: Button Enablement Precedes Component Rendering on Publish/Save Click

## What does this PR do?

Fixes the issue where clicking on the "Publish" or "Save" button results in the button transitioning to an enabled state before the associated component has fully rendered

Fixes #1399 

Loom video - https://www.loom.com/share/0d0cff1f53f14ac189498d8c712ef940?sid=13d6b767-c4f5-47f4-a779-f57ef81bffd3

## Type of change


## How should this be tested?

1. Navigate to the page where the "Publish" or "Save" button is located.
2. Ensure that the page is in a state where the button is initially disabled.
3. Click on the "Publish" or "Save" button.
4. Verify that the button transitions to an enabled state only after the associated component has fully rendered.
5. Ensure that this behavior is consistent on multiple attempts.

## Checklist

### Required

- [ ] First PR at Formbricks
